src/views/MultiTimeseries/MultiTimeseriesView.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent, useMemo } from 'react';
+import { MultiTimeseriesViewData } from './MultiTimeseriesViewData';
+import ViewWrapper from './ViewWrapper';
+
+type Props = {
+    data: MultiTimeseriesViewData
+    width: number
+    height: number
+}
+
+const MultiTimeseriesView: FunctionComponent<Props> = ({data, width, height}) => {
+    const divStyle: React.CSSProperties = useMemo(() => ({
+        width,
+        height,
+        overflowX: 'hidden',
+        overflowY: 'auto'
```

Comment:
I think it might be easier to trace things out later if these two static properties were set by a CSS class (something like `.MultiTimeseriesViewWrapperDiv` or whatever) rather than coming from code.

Reply:
yeah that would be the cleaner way to do it.